04-spatial-operations.qmd

```diff
@@ -225,9 +234,13 @@ The result is a boolean `Series`. Its contents should match your intuition: posi
 
 Again, the above output is a `Series` where each value represents a feature in the first input (`points`).
 In case we need to obtain a matrix of *pairwise* results, we can use `.apply` as follows. 
-In this case, the result is a `DataFrame`, where each row represents a `points` geometry and each column represents a `poly` geometry (there is just one, in this case)L
+In this case, the result is a `DataFrame`, where each row represents a `points` geometry and each column represents a `poly` geometry. We'll add another polygon to demonstrate:
```

Comment:
I think it's clearer if we add another polygon here

Reply:
Agree.